src/afl-analyze.c

```diff
@@ -989,6 +993,20 @@ int main(int argc, char **argv_orig, char **envp) {
 
         break;
 
+      case 'A':                                           /* CoreSight mode */
+
+        if (cs_mode) { FATAL("Multiple -A options not supported"); }
+
+        if (!(__aarch64__ && __linux__)) {
```

Comment:
this has to be an #if

Reply:
Oops, sorry, commit feff819 should fix this.